File: java/simonsays.java
import java.util.Scanner;

public class simonsays {
	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);
		int N = scan.nextInt();
		scan.nextLine();
		while (N-- > 0) {
			String command = scan.nextLine();
			if (command.startsWith("Simon says "))
                System.out.println(command.substring(11));
		}
		scan.close();
	}
}

File: ts/src/simonsays.ts
import readline = require('readline')

let lineCount = 0

readline
  .createInterface({
    input: process.stdin,
    output: process.stdout
  })
  .on('line', (line: string) => {
    if (lineCount > 0) {
      if (line.startsWith('Simon says ')) {
        console.log(line.substring(11))
      }
    }
    lineCount++
  })
  .on('close', () => {
    process.exit(0)
  })
